src/seeding.jl

```diff
@@ -53,8 +54,8 @@ function copyseeds!(S::DenseMatrix, X::DenseMatrix, iseeds::AbstractVector)
     return S
 end
 
-copyseeds(X::DenseMatrix{T}, iseeds::AbstractVector) where {T} =
-    copyseeds!(Matrix{T}(undef, size(X,1), length(iseeds)), X, iseeds)
+copyseeds(X::AbstractMatrix{<:Real}, iseeds::AbstractVector) =
+    copyseeds!(Matrix{eltype(X)}(undef, size(X,1), length(iseeds)), X, iseeds)
```

Comment:
```suggestion
    copyseeds!(similar(X, size(X, 1), length(iseeds)), X, iseeds)
```

Reply:
Then also stop using `DenseMatrix` in `copyseeds!`.